### DISCUSSION TITLE
Allow to set the label for pagination for better A11Y support

### DISCUSSION THREAD

**Discussion by hardikpthv** [ing-bank/lion#1966] [General]
Currently, in the pagination component, the `aria-label` is assigned from the static translations and it works as expected however when there is more than one pagination on the page, for example, one as part of the table (navigator) and another could be for perhaps page navigation. The screen reader will have the same label for both pagination components. 

https://github.com/ing-bank/lion/blob/master/packages/ui/components/pagination/src/LionPagination.js#L301

**Comment by gerjanvangeest** [comment]
Hi @hardikpthv, 

Yes, it would be good to add an option to set that label by the user. Feel free to create a PR 💪!
